### DISCUSSION TITLE
Autopilot Problems

### DISCUSSION THREAD

**Discussion by Amel170278** [flybywiresim/aircraft#3915] [Help]
Hi,
after the new Patch, when i insert a SID in the Flight Plan, the Autopilot don´t fly the Waypoints in NAV-Mode. Without a SID the Autopilot works normally. 
I create a Flight Plan in SimBrief.
Has anyone the same Problem?

**Comment by Paytheon** [comment]
I to am having issues flying with a simbrief flight plan. My issue is I never see PERF move off TAKEOFF page.

**Reply by BillyGeoThink2020** [reply]
I am working on it I apologize for the inconvenience. But need help on PFRE

**Reply by Amel170278** [reply]
? What ist PFRE?

**Reply by Hansvanwijhe** [reply]
I had the same problem, also many waypoints in the SImbrief generated flight plan were gone up in smoke. With the latest dev mod the problem seems to be gone.

**Reply by Amel170278** [reply]
I tried now with the dev Version and after take of i switch the Autopilot on the Plane turns into 360 Heading

**Reply by Paytheon** [reply]
This issue is being worked on. See this issue: https://github.com/flybywiresim/a32nx/pull/3924

**Reply by Amel170278** [reply]
Ok, thx

**Reply by Amel170278** [reply]
Its still not working with both mods. I also tried a different Flightplan from Simbrief. Its still the same. The Autopilot turns, after Take of to Heading 360. He not fly to the next Waypoint.

**Comment by Goose35211** [comment]
After the patch update once I toggle the AP the plane heads to a 360' and will not follow flight plan in FMC.  What is wrong with this?

**Reply by robinottawa** [reply]
Did you find a fix? When that happens to me I reenter the next waypoint using DIRect and it settles in to the route again. This doesn't work when in a STAR. You have to fly manually along the route until the ILS.

**Comment by Viopassch** [comment]
You need to set higher deadzones on your control axis. I can tell they're overriding the AP with drifting because the FD lines are not being followed.

**Reply by Goose35211** [reply]
No that’s not it. It has to do with the patch. It’s not a drift from FD lines it’s a drastic turn to 360’ as soon as I engage the Autopilot. It doesn’t matter what direction I am heading. I can fly manual along the FD indicator but as soon as I engage AP, automatic turn to 360’.

**Reply by Amel170278** [reply]
I have the same Problem as Goose35211. I tried the Deadzone Workaround. Its not working. The Autopilot without the Mod works normal.

**Comment by Amel170278** [comment]
Sometimes it works with the Deadzone Workaround when im starting on Runway (Eddf), sometimes not, in cold mode or on the Runway (Eddn). I Tried with the new exp. Mod....

**Comment by Amel170278** [comment]
Ok, after i deinstalled all Programms from Navigraph, it all works normal and fine.

**Reply by Goose35211** [reply]
what do you mean deinstalled all Programms from Navigraph?  What programms?

**Reply by Amel170278** [reply]
I thought Navigraph was the Problem. It was not. The Autopilot works now with the Dev Mod. I meant simlink, Navicharts and the Airac from Navigraph.
